components/About/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.title {
+    display: flex;
+    align-items: center;
+    margin: 15px;
```

Comment:
Make sure you only use `margin` with one value when you need equal margins all around. In this case you would just need `margin-bottom`, otherwise the title gets unaligned from the image.

![image](https://user-images.githubusercontent.com/15239884/114423715-4d43e300-9b6c-11eb-8409-17aa270d2ab6.png)